app/src/main/java/openfoodfacts/github/scrachx/openfood/utils/Utils.java

```diff
@@ -355,7 +355,11 @@ public static int getNovaGroupDrawable(@Nullable String novaGroup) {
     }
 
     public static int getSmallImageGrade(Product product) {
-        return getSmallImageGrade(product == null ? null : product.getNutritionGradeFr());
+        String nutritionGrade = null;
+        if (product != null) {
+            nutritionGrade = product.getNutritionGradeFr() != null ? product.getNutritionGradeFr() : product.getNutritionGradeTag();
```

Comment:
Why do we prefer the FR grade? Any particular reason? @teolemon @Varsha-29

Reply:
That's the name we initially gave to the Nutri-Score when it didn't have a name.

Reply:
Ok, but here we prefer using the FR tag. Are the FR tag and the "global" the same? Any differences?